src/components/Footer/index.js

```diff
@@ -1,19 +1,19 @@
 import React from 'react';
 import { Row, Column, Heading, Link, Paragraph } from '@datapunt/asc-ui';
-import styled from 'styled-components';
+import styled, { ascDefaultTheme } from '@datapunt/asc-core';
 
 const Disclaimer = styled.div`
   background-color: #666;
   padding-top: 20px;
   padding-bottom: 20px;
 
   * {
-    color: white;
+    color: ${ascDefaultTheme.colors.bright.main};
```

Comment:
Personally, I wouldn't use this reference for a simple a colour as 'white'. The indicator `bright.main` is very generic; a bit too generic to my liking. It might change over time and when it does, we will have to change our code.

Reply:
gedaan